code/modules/mob/living/simple_animal/friendly/spiderbot.dm

```diff
@@ -178,7 +178,7 @@
 
 /mob/living/simple_animal/spiderbot/proc/on_death()
 	UnregisterSignal(src, COMSIG_MOB_DEATH)
-	gib()
+	dust()
```

Comment:
dust is intended to be fully unrevivable

Reply:
Gib doesnt drop the brain anyway due to other issues with the mmi handling code. This just makes it look less weird until I figure out how to rewrite the whole mmi stuff.